controllers/release/adapter.go

```diff
@@ -478,3 +492,43 @@ func (a *Adapter) syncResources() error {
 
 	return a.syncer.SyncSnapshot(snapshot, releasePlanAdmission.Namespace)
 }
+
+// registerAttributionData updates the status of the Release being processed with the proper attribution author.
+func (a *Adapter) registerAttributionData(releasePlan *v1alpha1.ReleasePlan) error {
+	if a.release.IsAttributed() {
+		return nil
+	}
+
+	var author string
+	patch := client.MergeFrom(a.release.DeepCopy())
+
+	if a.release.Labels[metadata.AutomatedLabel] == "true" {
+		author = releasePlan.Labels[metadata.AuthorLabel]
+		if author == "" {
+			return fmt.Errorf("no author in the ReleasePlan found for automated release")
+		}
+		a.release.Status.Attribution.StandingAuthorization = true
+	} else {
+		author = a.release.Labels[metadata.AuthorLabel]
+		if author == "" { // this should be impossible
```

Comment:
I would rewrite the comment. As you said it could happen. Just say something like `webhooks prevent this to happen but they could be disabled in some scenarios`. Or something like that.

Reply:
done